tests/sdk/go/helm-local/step2/main.go

```diff
@@ -24,6 +24,15 @@ func main() {
 			})
 		ctx.Export("svc_ip", svc)
 
+		_, err = corev1.NewConfigMap(ctx, "cm", &corev1.ConfigMapArgs{
```

Comment:
Not entirely sure I caught what if anything was changed between step1 and step2? Granted this already existed outside of this change. Could we clarify what the intent with the additional step is?

Reply:
step1 tests the v2 Chart and step2 tests the v3 Chart.